### ISSUE TITLE
[feature request] Ignore specific versions

### ISSUE THREAD

**Issue by ktosiek** [pyupio/safety#20]
It would be nice to be able to opt out of specific affected versions - for example, Django 1.8.16 has vulnerabilities that a lot of users won't be affected by.
I'd like to ignore this specific vulnerabilities, but still get informed about others when they are added to safety-db.

**Comment by jayfk** [comment]
how would this work, exactly?

`safety check --ingore Django==1.8.16`?

**Comment by ktosiek** [comment]
Either that, or pointing to a file with a list of them.
It would be even better if there was a way to ignore specific vulns (so new vulnerabilities in the same version are still shown), but they don't seem to have any identifiers.

**Comment by jayfk** [comment]
The next `safety-db` release will have identifiers 👍

**Comment by jayfk** [comment]
This is now possible (https://github.com/pyupio/safety/commit/4506f379362754f4d312330e058cd05570ec4137), available in `1.4.0`.

### `--ignore`, `-i`

Ignore one (or multiple) vulnerabilities by ID

Example

```
safety check -i 1234
safety check --ignore=1234
safety check -i 1234 -i 4567 -i 89101
```
